airbyte-integrations/connectors/source-kobotoolbox/source_kobotoolbox/helpers.py

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+class Helpers(object):
+    @staticmethod
+    def get_json_schema():
+        json_schema = {}
+        return json_schema
```

Comment:
Why not move this to `source.py` there isn't any other function or file using it.